src/guards.ts

```diff
@@ -1,16 +1,23 @@
-import { FLUENTBIT_REGEX, COMMANDS, type FluentBitSchemaType, EXCLUDED_TAGS } from './constants';
+import {
+  FLUENTBIT_REGEX,
+  COMMANDS,
+  type FluentBitSchemaType,
+  EXCLUDED_TAGS,
+  FLUENTBIT_INCLUDE_REGEX,
+} from './constants';
 
-export const isFluentBit = (config: string) => !!config.match(FLUENTBIT_REGEX);
+export const isFluentBit = (config: string) =>
+  !!(config.match(FLUENTBIT_REGEX) || config.match(FLUENTBIT_INCLUDE_REGEX));
 
-export const isValidFluentBitSchemaType = (schema?: FluentBitSchemaType | null): schema is FluentBitSchemaType =>
+export const isValidFluentBitSection = (schema?: FluentBitSchemaType | null): schema is FluentBitSchemaType =>
   !!schema && !EXCLUDED_TAGS.has(schema.command.toLowerCase());
 
 export const isString = (value?: string): value is string => typeof value === 'string';
 
-export const isUsefulValidToken = (type?: string): boolean => isString(type) && !['space', 'comment'].includes(type);
+export const isUsefulToken = (type?: string): boolean => isString(type) && !['space', 'comment'].includes(type);
```

Comment:
Should these be some kind of enum literal rather than magic strings for `space` and `comment`?

Reply:
Yes! I will address shortly

Reply:
this is solved already, take a look at the latest commits.